docs/index.md

```diff
@@ -84,8 +84,18 @@ $ python setup.py install
 
 ### Command line completion
 
+#### Generating Completion Receipts
+
+The completion receipts for both Bash and ZSH can be generated by
+
+    make completion-scripts
+
+However, this receipt requires to have `watson` available in your PATH
+environment variable.
```

Comment:
I think auto-completion files should be distributed with Watson. One should update them while packaging a new release. Can you also update the contributing documentation [1] to add this step in the release process?

[1] http://tailordev.github.io/Watson/contributing/release-new-version/